### ISSUE TITLE
[Uri Bar] Error handling if Fetch timed out

### ISSUE THREAD

**Issue by egekorkan** [tum-esi/wade#40]
from Readme.md:
```
(+) Uri Bar: Fetch error handling when connection timed out
```

# Ideas:
* retry for **x** times
* stop trying and display error message if timed out always

**Comment by egekorkan** [comment]
I would suggest to not try to handle it. Since the user would be waiting on this screen anyways, he/she can click multiple times.

**Comment by egekorkan** [comment]
Thats a good point, still adding some Feedback that the connection timed out seems to be useful

**Comment by Fady222** [comment]
Basic handling is implemented in #98
